src/logger/src/metrics.rs

```diff
@@ -570,6 +570,49 @@ pub struct FirecrackerMetrics {
     pub uart: SerialDeviceMetrics,
     /// Metrics related to signals.
     pub signals: SignalMetrics,
+    /// Metrics related to virtio-vsockets.
+    pub vsock: VsockDeviceMetrics,
+}
+
+/// Vsock-related metrics.
+#[derive(Default, Serialize)]
+pub struct VsockDeviceMetrics {
```

Comment:
nit: would be nice to move this struct before `struct SerializeToUtcTimestampMs;`.

Reply:
Done